### PR TITLE
Make quanta os discovery more specific

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: includes/definitions/quanta.yaml [hunk 1]
```diff
@@ -6,9 +6,10 @@ over:
...
-    - sysDescr:
-        - vxworks
-        - quanta
-        - LB9, Runtime Code
+          - .1.3.6.1.4.1.7244
+    -
+      sysObjectId:
+          - .1.3.6.1.4.1.4413
+      sysDescr_regex:
+          - '/vxworks/i'
```
Comment (murrant) [inline includes/definitions/quanta.yaml]: Still unhappy about the case insensitivity here...

Comment (laf) [inline includes/definitions/quanta.yaml]: I know you are :) But, we don't know their devices don't have varying case sensitivity. We've not yet come across devices being detected as Quanta.

Comment (murrant) [inline includes/definitions/quanta.yaml]: We do have the one confirmed output from @tobiasa1 which validates VxWorks and the snmprec files we have look real.

Comment (f0o) [inline includes/definitions/quanta.yaml]: LB6M:
```
.1.3.6.1.2.1.1.1.0 = STRING: LB4M 48x1G 2x10G, 1.1.0.8, VxWorks 6.6
.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.4413
```

LB9M will follow on monday morning when I turn it on

Comment (laf) [inline includes/definitions/quanta.yaml]: Updated

**Comment by murrant** [comment]
@f0o It only matches sysDescr if sysObjectID matches `.1.3.6.1.4.1.4413`

**Comment by f0o** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

:+1:
-----BEGIN PGP SIGNATURE-----

iQJPBAEBCgA5MhxEYW5pZWwgUHJldXNza2VyIChNb2JpbGVLZXkjMikgPGYwb0Bs
aWJyZW5tcy5vcmc+BQJYePN+AAoJEJMZKtVWOFzB8r0P/1Z0rIMKDOrF1tHGusMQ
rpyLZlIWBtk2sr1pODd40HmC4Z/E3q6BM88aHiZToHIVMf2Q5HugiFEzwEdCiWyS
afP/vrFwIMQ+DG3b2jWbc4e3VQDbQjbEx8CMJbK7bPUq8ermS/GpZEj9pUiOAm5W
j6ikvLHOcdbmK2CqNvEBEJHfCeq95ShPdWxKAyOdxf9jIWIoWCgW5nsOWHXU0Som
GGPqc1r5NzgWSEIhBMvCpbA2Gw+OiyMxBBngtc/9JvnJkpVBEsJmgS1npIz+3KJj
8MpT0cXIHdhqqVRbV43RWjkZrhLojfG8646E+FCarJOS7Lr/lbB+UwKSWiMOrwzw
VBu4K2VxjhNMjYMkwmfEyx/+7mr5ENqFgObuFLuOH8euq3+BQYkFflRL+HTHKFUw
KVvv+zZrXg/UxUC5vaYxgSrrg04/DQGjprP++0Gi7BYNVKJ7yvm+4rv63noPAKDG
E3vYvJeLImPzWyoiKakxZv4T2hHCY6Qsvu4sVZ4D/3mie6ZQmziRR+RCTKpPRdLg
S9rOtCJmVqoxIbYVVz41tBlSykFXOpF03j0AkJlXIr7+BeS2YJBeeRrkJ/YOvVZa
YE9MXuwZyV8xkf/NWOvee1iLDYFyAL7ulWIbMfGAn/G2w5Y1KLtAH6Cg11iJSU00
yWU5WnaZ4NAIzJBumKJpDIR9
=hoM2
-----END PGP SIGNATURE-----

**Comment by murrant** [comment]
@f0o if you find that additional info, we can merge another change :-)

**Pull request merged**